form_designer/migrations/0001_initial.py

```diff
@@ -4,7 +4,11 @@
 from django.db import migrations, models
 from django.conf import settings
 import form_designer.fields
-import picklefield.fields
+from form_designer.settings import VALUE_PICKLEFIELD
+if VALUE_PICKLEFIELD:
+    from picklefield.fields import PickledObjectField as data_field_type
+else:
+    data_field_type = models.TextField
```

Comment:
Not sure if this will work, but maybe we could just drop that feature of being able to change the field type from settings. I guess none of our projects use that anyway, but rather use the default (PickledObjectField).

So I'm proposing to just remove the VALUE_PICKLEFIELD setting (or hard-code it to True) and then hacks like this could be avoided.